addons/resource.language.en_gb/resources/strings.po

```diff
@@ -12222,7 +12225,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#23050"
-msgid "Activate teletext"
+msgid "Activate Teletext"
```

Comment:
Yes, however I was already told off its *Teletext* earlier on, even though its not on external references.. Same for Zeroconf...

> @MartijnKaijser wrote
> Teletext is a name for a service. Hence it should be capitalized

The general rule seems to indicate as long as any given program refers to something consistently throughout, that it should be acceptable. So I didnt mind changing it to Capitalized word.

If you guys can Agree on  Teletext <-> teletext and Zeroconf <-> zeroconf I'm OK with going along here.